app/pom.xml

```diff
@@ -38,6 +45,93 @@
     </dependency>
   </dependencies>
 
-  <!--  TODO jib-->
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-api</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>com.expediagroup.streamplatform</groupId>
+                  <artifactId>stream-registry-graphql-api</artifactId>
+                  <version>${project.version}</version>
+                  <includes>*.graphql</includes>
```

Comment:
Could you explain why you need to make sure the `.graphql` file is included in the `app` jar?

Reply:
I thought the *.graphql file is necessary for springboot during initialization so that it knows what to bind.

Reply:
Without this, it complains that no graphql files are on the classpath

Reply:
I tried this locally and it seemed to work ok. The `graphql-api` contains this resource and is on the classpath at runtime.

Reply:
Removing this seems to work now... I know it didn't with the spotify docker-maven-plugin